Cocchicheck: fix docker container package issues

**Describe the bug**

> 08:51:39 D patchwise.dockermanager docker.py#273: Executing command in container: docker exec --workdir /home/patchwise/build patchwise-coccicheck-4414422a92cc7f2329608a7ccc6e07a3d5a56832 make -C /home/patchwise/kernel O=/home/patchwise/build -j16 ARCH=arm64 LLVM=1 -s coccicheck M=arch/arm64/boot/dts/qcom MODE=report DEBUG_FILE=/tmp/patchwise.patch_review.static_analysis_null
> 08:51:39 D patchwise.coccicheck patch_review.py#117:
> Please check for false positives in the output before submitting a patch.
> When using "patch" mode, carefully review the patch before submitting it.
> 
> coccicheck failed
> 
> 08:51:39 D patchwise.coccicheck patch_review.py#120: make[2]: *** [/home/patchwise/kernel/Makefile:2239: coccicheck] Error 255
> make[1]: *** [/home/patchwise/kernel/Makefile:248: __sub-make] Error 2
> make: *** [Makefile:248: __sub-make] Error 2
> 
> 08:51:39 D patchwise.coccicheck patch_review.py#127: coccicheck running... 0s elapsed
> 08:51:39 D patchwise.coccicheck coccicheck.py#107: Coccicheck output for arch/arm64/boot/dts/qcom:
> 
> Please check for false positives in the output before submitting a patch.
> When using "patch" mode, carefully review the patch before submitting it.
> 
> coccicheck failed
> make[2]: *** [/home/patchwise/kernel/Makefile:2239: coccicheck] Error 255
> make[1]: *** [/home/patchwise/kernel/Makefile:248: __sub-make] Error 2
> make: *** [Makefile:248: __sub-make] Error 2
> 

**To Reproduce**
Steps to reproduce the behavior:
Run:
```
patchwise --repo-path path/to/kernel --reviews Coccicheck --log-level DEBUG
```

**Expected behavior**
Coccicheck should not error out.


**Additional context**
The likely root cause is missing `ocaml-native-compilers` package install in the docker image/container.

--- a/patchwise/dockerfiles/Coccicheck.Dockerfile
+++ b/patchwise/dockerfiles/Coccicheck.Dockerfile
@@ -4,8 +4,10 @@ FROM patchwise-base:latest
 USER root
 
 # Install coccinelle
+# Install OCaml native compiler (it is used for some semantic patches)
 RUN apt-get update && apt-get install -y --no-install-recommends \
     coccinelle \
+    ocaml-native-compilers \
     && rm -rf /var/lib/apt/lists/*
 
 USER patchwise

